### PR TITLE
Add unit test target

### PR DESCRIPTION
**Pull request by baijum**
Run unit tests in verbose mode.

### CODE REVIEW

File: ci-operator/jobs/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -107,3 +107,46 @@ presubmits:
...
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: redhat-developer-service-binding-operator-master.yaml
+              name: ci-operator-master-configs
+        image: ci-operator:latest
+        imagePullPolicy: Always
+        name: ""
```
Comment (DhritiShikhar) [inline ci-operator/jobs/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master-presubmits.yaml]: Does name have to be an empty string

Comment (baijum) [reply ci-operator/jobs/redhat-developer/service-binding-operator/redhat-developer-service-binding-operator-master-presubmits.yaml]: @DhritiShikhar  This is an auto-generated file. Not sure about it.

**Pull request merged**